script.js

```diff
@@ -1,28 +1,31 @@
-const msgWaiting = document.querySelector(".message-waiting");
-const searchForm = document.getElementById("search-form");
-const booksDisplay = document.getElementById("books-display");
+const $msgWaiting = $(".message-waiting");
+const $searchForm = $("#search-form");
+const $booksDisplay = $("#books-display");
 
 const googleBooksURL = "https://www.googleapis.com/books/v1/volumes?q=";
 const apiKey = "&key=AIzaSyCw6aeDwd2srAlaiVBFSSBvyl2O7atCTaA";
 
-searchForm.addEventListener("submit", () => {
-  msgWaiting.style.display = "none";
-  const searchInput = document.getElementById("search-input").value;
+$searchForm.on("submit", () => {
+  $msgWaiting.hide();
+  const searchInput = $("#search-input").val();
   const searchQuery = `${googleBooksURL}${searchInput}${apiKey}`;
 
   getBooks(searchQuery);
 });
 
-async function getBooks(query) {
-  const booksPromise = await fetch(query);
-  const booksData = await booksPromise.json();
-
-  displayBooks(booksData.items);
+function getBooks(query) {
+  $.ajax({
+    url: query,
+    method: "GET",
+    datatype: "json"
+  }).done((booksData) => {
+    displayBooks(booksData.items);
+  });
```

Comment:
This is the core part of this demo: the difference between the newer fetch API, and the more legacy (but still used) jQuery + Ajax approach.

When making an Ajax request, you don't need to specify the method if you're making GET requests, or the datatype because jQuery will (usually) infer it on its own. But it doesn't hurt to have a bit more clarity and resilience.

Note also that the `ajax` function returns a Promise, so we can use the Promise API to bind a callback for what to do once the Ajax request is successful.